### ISSUE TITLE
Fix rickshaw bower workaround

### ISSUE THREAD

**Issue by findkiko** [DctrlVan/numerator#25]
TLDR; Add [this](https://gist.github.com/81a34998c9b02b5d2ae9) file to 'bower_components/rickshaw' directory and re-issue `gulp serve` command.

`bower install rickshaw` works but rickshaw has no proper bower.json file that gulp/wiredep uses to install the right scripts at build time. This is fixed by manually adding a bower.json into `bower_components/rickshaw` directory. This is a (now-closed) bug on rickshaw: https://github.com/shutterstock/rickshaw/issues/361. Although the file still has no bower.json file, I believe other users install the package witha yeoman generator which does this step for them.

**Comment by findkiko** [comment]
Elders advise a switch to Browserify framework. For the meantime this is reduced to a standard copy of package.json to bower.json within rickshaw folder by commit 27a41245.
